src/components/WorkflowSelection.jsx

```diff
@@ -143,10 +147,12 @@ const WorkflowSelection = ({ className, dispatch, location, history, translate,
               <span className="primary-label">{translate('workflowSelection.keywordSearch')}</span>
               <div className="disabled-workflow">
                 <a
-                  className="tertiary-label"
-                  data-disabled-href={`${classifyPath}${c.arabicKeyword}`}
-                  data-disabled-target="_blank"
-                  href="#"
+                  className={classnames('tertiary-label', {
+                    'disabled-workflow--enable': adminMode
+                  })}
+                  href={`${classifyPath}${c.arabicKeyword}`}
+                  rel="noopener noreferrer"
+                  target="_blank"
```

Comment:
Minor note: ditto on the 'disabled' `<a>` tag here being accessible via keyboard.